docs/video-miners/how-to-guides/l2-migration.md

```diff
@@ -43,26 +43,22 @@ Other options are: [set up an Arbitrum node](https://developer.offchainlabs.com/
     
 2. **Follow the prompt to begin migration to Arbitrum**
```

Comment:
Maybe change this to "Navigate to the L2 Migration Tool to begin migration to Arbitrum". The prompt will only appear if they're wallet is connected to rinkeby